src/git.md

```diff
@@ -263,6 +264,26 @@ You also may want to squash just the last few commits together, possibly
 because they only represent "fixups" and not real changes. For example,
 `git rebase --interactive HEAD~2` will allow you to edit the two commits only.
 
+### `git range-diff`
+
+After completing a complicated rebase, or even a relatively simple one, you may
+want to review the changes between your old branch and your new one. You can do
+that with the `git range-diff master @{u} HEAD` command, which can be very
```

Comment:
Maybe say that `@{u}` means the upstream branch? I had to figure it out by experimenting.

Reply:
I added a bunch more detail, let me know what you think.